JS/see_article.js

```diff
@@ -172,8 +192,25 @@ function create_see_article(article) {
         }
         if (article_already_in_cart == false) {
             tab.push({ 'id': article.id, 'name': article.name, 'price': article.price, 'brand': article.brand, 'description': article.description, 'preview': article.preview, 'quantity': quantity_article });
+            document.getElementById("nb_article_in_cart").textContent = tab.length;
         }
         sessionStorage.setItem('articleToCart2', JSON.stringify(tab));
+        //Soit on affichera directement la pannier pour indiquer l'ajout soit on devra avoir un pop up
+       // location.href = "cart.html";
+
+        document.getElementById("overlay_alert").style.display = "block";
+        document.getElementById("popupAlert").style.display = "block";
+        myVar = setTimeout(alertClosed, 1500);
+
+
+    
+        
+
     }
     return box_article
+}
+
+function alertClosed() {
```

Comment:
it looks like this function is only called once and with a setTimeout. does this mean that the popup can't be closed without waiting ?

Reply:
Yes. Do you think it's better to put a button? If I put one, each addition to the cart will force the user to close the window by himself without automation.

Reply:
You can add a button and let the timeout. You will just need to pay attention to edge case like if the user add an item to the cart, close the popup, and add another item in less than the timeout.

Reply:
Done.